target/scripts/startup/setup.d/security/rspamd.sh

```diff
@@ -283,6 +284,21 @@ function __rspamd__setup_hfilter_group() {
   fi
 }
 
+
+# This function handles setup of the neural module (see
+# https://www.rspamd.com/doc/modules/neural.html). This module is experimental 
+# but can enhance anti-spam scoring possibly.
+function __rspamd__setup_neural() {
+  if _env_var_expect_zero_or_one 'RSPAMD_NEURAL' && [[ ${RSPAMD_NEURAL} -eq 1 ]]; then
+    __rspamd__log 'debug' 'Disabling Neural module'
```

Comment:
Sorry, I made a wrong suggestion... I will apply immediately

```suggestion
    __rspamd__log 'debug' 'Enabling Neural module'
    __rspamd__log 'warn' 'The Neural module is still experimental (in Rspamd)'
```